packages/blocks/calculation/package.json

```diff
@@ -0,0 +1,122 @@
+{
+  "name": "@hashintel/calculation",
+  "version": "0.2.0",
+  "description": "Spreadsheet-like table for doing computation on entities",
+  "keywords": [
+    "blockprotocol",
+    "blocks"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hashintel/hash.git",
+    "directory": "packages/blocks/calculation"
+  },
+  "license": "MIT",
+  "author": "Ahmad Sattar Atta",
+  "scripts": {
+    "build": "node_modules/.bin/block-scripts build",
+    "dev": "node_modules/.bin/block-scripts dev",
+    "fix:eslint": "eslint --ext .ts,.tsx --fix ./src/",
+    "lint:eslint": "eslint --ext .ts,.tsx ./src/",
+    "lint:tsc": "tsc --noEmit",
+    "serve": "node_modules/.bin/block-scripts serve"
```

Comment:
```suggestion
    "build": "block-scripts build",
    "dev": "block-scripts dev",
    "fix:eslint": "eslint --ext .ts,.tsx --fix ./src/",
    "lint:eslint": "eslint --ext .ts,.tsx ./src/",
    "lint:tsc": "tsc --noEmit",
    "serve": "block-scripts serve"
```

Reply:
Oh, i see the other blocks do the same. Is this specific to our monorepo setup, or could this be changed in the react template as well?

Reply:
Not sure why we'd need`node_modules/.bin` prefix in both cases. Mentioning a dependency via `yarn xyz` should resolve the binary. There is no guarantee that the file will be in `node_modules/.bin` in a general case scenario.